README.md

```diff
@@ -8,149 +8,193 @@ Peter Rushforth
+MapML is the declarative API that results from our community’s effort to abstract 
+and polyfill Web mapping requirements as custom elements, using one of the mature 
+open source Web mapping frameworks so as to not have to start “from scratch”.  
+The process of polyfilling Web maps as MapML has also identified missing low-level 
+platform APIs that will be essential to complete the implementation of Web maps 
+as a standard platform feature.
 
 <h2>Contents</h2>
 
 - [The Problem](#the-problem)
 - [The Proposal](#the-proposal)
-  - [Goals](#goals)
-  <!-- - [Non-Goals](#non-goals) -->
-  - [A High-Level API](#a-high-level-api)
-  - [Key Scenarios](#key-scenarios)
+- [Goals](#goals)
+- [Use Cases and Requirements](#use-cases-and-requirements)
+- [Key Scenarios](#key-scenarios)
 - [Detailed design discussion](#detailed-design-discussion)
-  - [Use Cases and Requirements](#use-cases-and-requirements)
-  - [W3C/OGC Joint Workshop on Maps for the Web](#w3c-ogc-joint-workshop-on-maps-for-the-web)
 - [Considered Alternative Designs of MapML](#considered-alternative-designs-of-mapml)
-- [Alternative Approaches](#alternative-approaches)
 - [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
 - [References and Acknowledgements](#references-and-acknowledgements)
 
 <h2 id="the-problem">The Problem</h2>
 
-Web maps today are created using a wide range of technology stacks on both the client and server, some standard, some open, and some proprietary.  The complexity of choices and the wide variety of technologies required to create Web maps results in <a href="https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md">maps of highly variable usability and accessibility</a>.  This has in turn led to the creation of centralized mapping services, that may or may not be implemented using Web technology; in some cases, mapping services which work well on desktop Web browsers mostly bypass the mobile Web through creation of mobile platform mapping apps, where the ‘rules of the Web platform’ (such as device permissions) do not apply.  Some centralized mapping services, both on the Web but especially on mobile technology platforms, are constructed for the purpose of tracking the user’s location and their locations of (search) interest, and using that private location information to market and re-sell highly targeted advertising. 
-
-The problem to be solved, therefore, is to reduce the threshold complexity of creating accessible, usable and privacy-preserving Web maps, and to enable full use of Web platform standards such as HTML, URL, SVG, CSS and JavaScript in map creation, styling, presentation and interaction. 
+Web maps are <span style="text-decoration:underline;">not interoperable</span>. 
```

Comment:
`<span style="text-decoration:underline;">` doesn't work. <ins>You should be able to use `<ins>` instead</ins>.